Makefile

```
@@ -4,6 +4,11 @@ ROX_PROJECT=apollo
 TESTFLAGS=-race -p 4
 BASE_DIR=$(CURDIR)
 
+DOCKER_VERSION=$(shell docker --version)
```

```suggestion
DOCKER_VERSION=$(shell docker --version 2>/dev/null)
```
To hide the errors that you found when `make` is running inside the container.